### DISCUSSION TITLE
[CONCEPT] Pair/match the same amount of lines from one file to other

### DISCUSSION THREAD

**Discussion by EloyFarina** [SubtitleEdit/subtitleedit#8696] [Ideas]
The idea is use something similar to "Point sync via another subtitle" feature.
On left side, the file to be edited; on right side, the source file.
The goal is match the same amount of lines from one file to other.
As I couldn't find any IA that makes this automatically, the idea is do it manually for now.

This is just a concept that I'm looking for:
![SUBIR 1](https://github.com/user-attachments/assets/7f4cbc27-fc12-4549-8b8e-6f614264274c)
As you can see, the first 5 lines are missing on left side, so I select them and click on "Copy lines and times to" button.
This is it should looks
![SUBIR 2](https://github.com/user-attachments/assets/19b11f77-6638-43b9-b21c-793a7b1fb270)

Then If I need to split some lines, i can right-click on the subtitle line and select Split lines from there
![SUBIR 3](https://github.com/user-attachments/assets/949f1066-a241-4e6a-8026-e9b4a9cc4b90)

And after match all lines manually, i can click on "Copy whole times to" button.
![SUBIR 4](https://github.com/user-attachments/assets/c4e56f5d-6c04-4282-90f3-90552e17a107)
At that way I will have the same ammounts of lines with the correct timestamps from other subtitle.

Again, I would like an IA that makes this job, but as so far there is nothing, It would be great if we can do it manually meanwhile.

Regards

**Comment by Ra1nek** [comment]
**EloyFarina**, I missed your idea and proposed a similar one. Thank you for the concept with photos. I also suggested separate subtitle editing without linking to the dialogues. This feature, as well as synchronized scrolling of visual dialogues for line comparison, is missing in **Subtitle Edit**. However, I believe that the timing transfer, as I suggested in another idea, should be more flexible, allowing the selection and transfer of individual dialogues between subtitles, even if their numbers don't match. Essentially, this would be an editor for simultaneous, independent editing of two subtitles. Often, original subtitles in different languages are of poor quality, and editing them takes a lot of time before you can start working in your native language. Automation would be helpful, as while **Subtitle Edit** allows you to adjust everything to the audio track, it is very time-consuming.
